pipeline/realtime.py

```diff
@@ -0,0 +1,71 @@
+from celery import chain
+from orca.tasks.pipeline_tasks import (
+    flag_ants_task,
+    flag_with_aoflagger_task,
+    average_frequency_task,
+    applycal_data_col_task,
+    wsclean_task,
+    peel_with_ttcal_task,
+    change_phase_center_task
+)
+
+
+import glob
+import os
+
+# Specify the directory with ms files
```

Comment:
I'm paranoid about this so I usually start my scripts with `if __name__=='__main__`. If a worker tries to import this module it can go very wrong...